mne/_fiff/meas_info.py

```diff
@@ -3179,12 +3182,18 @@ def create_info(ch_names, sfreq, ch_types="misc", verbose=None):
 
     Proper units of measure:
 
-    * V: eeg, eog, seeg, dbs, emg, ecg, bio, ecog
-    * T: mag
+    * V: eeg, eog, seeg, dbs, emg, ecg, bio, ecog, resp, fnirs_fd_ac_amplitude,
+      fnirs_cw_amplitude, fnirs_od
```

Comment:
IDK why this is formatting as bold on the first line; one easy approach is to put it all on one line and add a `noqa` annotation at the end of the docstring

```suggestion
    * V: eeg, eog, seeg, dbs, emg, ecg, bio, ecog, resp, fnirs_fd_ac_amplitude, fnirs_cw_amplitude, fnirs_od
```

Reply:
It's not bold anymore:

https://output.circle-artifacts.com/output/job/da6cfdc8-65ab-4a73-b657-780162feebd3/artifacts/0/html/generated/mne.create_info.html

![image](https://github.com/mne-tools/mne-python/assets/2365790/79b0823e-461a-4424-af65-079dde26ac76)

It was bold before because the indentation was incorrect, see https://github.com/mne-tools/mne-python/pull/12509#discussion_r1543390376